src/pages/memories/MemoriesComponent.js

```diff
@@ -0,0 +1,60 @@
+import Header from "../../components/header/Header";
+import Footer from "../../components/footer/Footer";
+import { memories } from "../../portfolio";
+import React, { Component } from "react";
+import { Fade } from "react-reveal";
+import "./MemoriesComponent.css";
+
+const MemoriesData = memories.memoriesSection;
+
+class Memories extends Component {
+  render() {
+    const theme = this.props.theme;
+    return (
+      <div>
+        <Header theme={theme} />
+        <Fade bottom duration={2000} distance="20px">
+          <div className="heading-div">
+            <h1 className="headingtext" style={{ color: theme.text }}>
+              Some Precious Moments 😇{" "}
```

Comment:
For adding emoji you need to take care of some things. 
(1) You need to disable eslint after import like [this](https://github.com/ashutosh1919/masterPortfolio/blob/1f814af67dee3d74bfdba954eda49cd3104c4412/src/components/footer/Footer.js#L5).
(2) You need to wrap emoji into span tag like [this](https://github.com/ashutosh1919/masterPortfolio/blob/1f814af67dee3d74bfdba954eda49cd3104c4412/src/components/footer/Footer.js#L12)